Makefile

```diff
@@ -17,6 +20,7 @@ help:
 	@echo " * 'verify'        - Execute the source code verification tools"
 	@echo " * 'install.tools' - Installs tools used by verify"
 	@echo " * 'uninstall'     - Remove installed binaries from system locations"
+	@echo " * 'version'       - Print git tag version"	
```

Comment:
"git tag version" sounds a little bit weird to me. What about `Print current cri-containerd release version`?